crates/autopilot/src/solvable_orders.rs

```diff
@@ -489,7 +490,7 @@ fn get_orders_with_native_prices(
         }
     });
 
-    let tokens_by_priority = prioritize_missing_prices(filtered);
+    let tokens_by_priority = prioritize_missing_prices(&mut filtered);
```

Comment:
Why do we need to call this twice? Does anything change in between the two?

Reply:
Since we fetch a bunch of tokens from the first instance of `tokens_by_priority` it now contains tokens that are no longer relevant (i.e. no longer have to be prioritized). Alternatively we could also remove tokens we were able to fetch from the original `tokens_by_priority` but we'd have to use `shift_remove()` to not mess up the order (i.e. priority) which would be more expensive and more implicit than simply recomputing everything.